tasks/main.yml

```diff
@@ -106,6 +106,11 @@
     path={{ kube_manifests_dir }}
     state=directory
 
+- name: Copy vsphere.conf
```

Comment:
replace keyword vpshere with cloud-config in this task

Reply:
Done